### PR TITLE
Docstring runner loaders

### PR DESCRIPTION
**Pull request by diwei-tan**
Docstrings for runner and loaders. Closes issue #326

### CODE REVIEW

File: peekingduck/runner.py [hunk 1]
```diff
@@ -26,23 +26,41 @@
...
+        CUSTOM_NODE_PARENT_FOLDER (:obj:`str`): path to folder which contains \
+        custom nodes that users have created to be used with PeekingDuck. \
+        For more information on using custom nodes, please refer to \
+        `getting started <getting_started/03_custom_nodes.html>`_.
+
+        nodes (:obj:`list` of :obj:`Node`): if not using declarations via yaml, \
+        initialize by giving the node stack directly as a list.
+
+    Outputs:
+        None. Runs the pipeilne using Runner.run()
```
Comment (sidney-tio) [inline peekingduck/runner.py:55]:
```suggestion
        None. Runs the pipeline using Runner.run()
```

File: peekingduck/configloader.py [hunk 2]
```diff
@@ -23,8 +23,19 @@
 
 
 class ConfigLoader:  # pylint: disable=too-few-public-methods
-    """ Reads configuration and returns configuration required to
-    create the nodes for the project
+    """
+    Config loader class is a helper class to create pipeline.
```
Comment (sidney-tio) [inline peekingduck/configloader.py:27]:
```suggestion
    A helper class to create pipeline.
```

File: peekingduck/configloader.py [hunk 3]
```diff
@@ -41,8 +52,18 @@ def _get_config_path(self, node: str) -> str:
         return filepath
 
     def get(self, node_name: str) -> Dict[str, Any]:
-        """Get item from configuration read from the filepath,
-        item refers to the node item configuration you are trying to get"""
+        """
+        Function to get node configuration for specified node.
```
Comment (sidney-tio) [inline peekingduck/configloader.py:56]:
```suggestion
        Get node configuration for specified node.
```

File: peekingduck/declarative_loader.py [hunk 4]
```diff
@@ -35,7 +35,30 @@
 
 
 class DeclarativeLoader:  # pylint: disable=too-few-public-methods
-    """Uses the declarative run_config.yml to load pipelines or compiled configs"""
+    """
+    Declarative loader class is a helper class to create pipeline.
```
Comment (sidney-tio) [inline peekingduck/declarative_loader.py:39]:
```suggestion
    A helper class to create pipeline.
```

File: peekingduck/declarative_loader.py [hunk 5]
```diff
@@ -35,7 +35,30 @@
...
+
+    The declarative loader class creates the specified nodes according to any
+    modfications provided in the configs and returns the pipeline needed for
+    inference.
+
+    Args:
+
+        run_config (:obj:`str`): Path to yaml file of node pipeine declaration
```
Comment (sidney-tio) [inline peekingduck/declarative_loader.py:47]:
```suggestion
        run_config (:obj:`str`): Path to yaml file that declares the node sequence to be used in the pipeline
```

File: peekingduck/runner.py [hunk 6]
```diff
@@ -26,23 +26,41 @@
 
 
 class Runner():
-    """Runner class that uses the declared nodes to create pipeline to run inference
+    """
+    The runner class for creation of pipeline using declared/given nodes.
+
+    The runner node situationally, depending on different inputs, use the
+    provided configurations to setup a node pipeline whih is used to run single
+    or continuous inference.
```
Comment (sidney-tio) [inline peekingduck/runner.py:34]: This sentence is quite strange

File: peekingduck/configloader.py [hunk 7]
```diff
@@ -23,8 +23,19 @@
...
+    The config loader class is used to allow for instantiation of Node classes
+    directly instead of reading configurations from the run config yaml.
+
+    Args:
+
+        basedir (:obj:`str`): base directory of peekingduck
+
+    Outputs:
+        None
+
```
Comment (sidney-tio) [inline peekingduck/configloader.py:38]: _init__ does not need outputs

File: peekingduck/declarative_loader.py [hunk 8]
```diff
@@ -35,7 +35,30 @@
...
+        config changes passed as part of cli command. Used to modify the node \
+        configurations direct from cli.
+
+        custom_node_parent_folder (:obj:`str`): path to parent folder which contains \
+        custom nodes that users have created to be used with PeekingDuck. \
+        For more information on using custom nodes, please refer to \
+        `getting started <getting_started/03_custom_nodes.html>`_.
+
+    Outputs:
+        None. Creates pipeline by using DeclarativeLoader.get_pipeline()
```
Comment (sidney-tio) [inline peekingduck/declarative_loader.py:59]: __init__ does not need outputs

File: peekingduck/runner.py (hunk 1 above)
Comment (sidney-tio) [inline peekingduck/runner.py:55]: __init__ does not need outputs

File: peekingduck/configloader.py (hunk 7 above)
Comment (diwei-tan) [reply peekingduck/configloader.py:38]: Removed

File: peekingduck/declarative_loader.py (hunk 8 above)
Comment (diwei-tan) [reply peekingduck/declarative_loader.py:59]: Removed

File: peekingduck/runner.py (hunk 1 above)
Comment (diwei-tan) [reply peekingduck/runner.py:55]: Removed

File: peekingduck/runner.py [hunk 9]
```diff
@@ -29,9 +29,8 @@ class Runner():
     """
     The runner class for creation of pipeline using declared/given nodes.
 
-    The runner node situationally, depending on different inputs, use the
-    provided configurations to setup a node pipeline whih is used to run single
-    or continuous inference.
+    The runner node uses the provided configurations to setup a node pipeline
+    which is used to run single or continuous inference.
```
Comment (sidney-tio) [inline peekingduck/runner.py:33]: Dont think the runner can be considered a node right? 
```suggestion
    The runner class uses the provided configurations to setup a node pipeline
    which is used to run inference.
```

**Pull request merged**